tests/cli/test_get.py

```diff
@@ -165,3 +166,36 @@ def test_get_data_with_overwrite_false(self, mock_query_data):
             overwrite=False,
             interactive=False,
         )
+
+    # MARK : User Chooses to skip
+
+    @patch("scribe_data.cli.get.query_data")
+    @patch("scribe_data.cli.get.Path.glob")
+    @patch("builtins.input", return_value="s")
+    def test_user_skips_existing_file(self, mock_input, mock_glob, mock_query_data):
+        mock_glob.return_value = [Path("./test_output/English/nouns.json")]
```

Comment:
Thank you for the feedback! I appreciate the suggestion about adding docstrings. I’m still gaining experience in writing tests, and since the other tests in the file didn’t include docstrings, I assumed it wasn’t a priority.  Now I understand how important they are for clarity and maintainability, and I’ll make sure to add them moving forward.

I'll work on improving it!